smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/SmokeTestGenerator.kt

```diff
@@ -0,0 +1,267 @@
+package software.amazon.smithy.swift.codegen.integration
```

Comment:
Generates smoke tests according to generic Smithy spec. A child class in aws-sdk-swift extends this class to customize AWS specific logic by overriding vendor specific methods.